src/Model/ModelTrait/DBReadTrait.php

```diff
@@ -289,6 +291,16 @@ protected function setSet($ignoreNoRecord = false)
         }
 
         if ($selectStatement->rowCount()) {
+            if ($this->isIncludeTotalCount() == true) {
```

Comment:
This is to only include totalCount when query parameter `include_count=true` is set